content/topics/intro-to-typescript/_index.md

```diff
@@ -5,11 +5,28 @@ ready: true
 title: Introduction to Typescript
 ---
 
+### What is Typescript
+
+- TypeScript is an extension of JavaScript that adds types as well as other language features.
+- Typescript was created to make it easier to write code that is compatible with the JavaScript language.
+- As its name implies, adding types to JavaScript is the primary purpose of TypeScript, and it is the reason that people use it today.
+
+#### The TypeScript Compiler
+
+- Javascript is an interpreted language, meaning that it is not compiled to machine code like C++ or Java.
+- Therefore, it needs to be run to test its correctness, thats is why you spend more time trying to find bugs in your code.
```

Comment:
There's a grammar mistake `...that's is why ...`

Please review this line. It seems to be a continuation of the first sentence but also a separate point on it's own

You can either make the first point and the second point one sentence, or you can find a way to communicate two saperate points about the `TypeScript Compiler`